dnas/appstore/types/Cargo.toml

```diff
@@ -9,9 +9,9 @@ name = "appstore_types"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-hc_coop_content_sdk = "0.7"
-hc_crud_caps = "0.17"
-hdi = "=0.5.0-dev.12"
+hc_coop_content_sdk = { git = "https://github.com/holochain/hc-cooperative-content.git", package = "hc_coop_content_sdk" }
```

Comment:
Using git dependency for this reason https://github.com/holochain/portal-dna/pull/4

Can be changed once ownership is fully transferred